tutorials/navigation/navigation_using_navigationlinks.rst

```diff
@@ -0,0 +1,50 @@
+.. _doc_navigation_using_navigationlinks:
+
+Using NavigationLinks
+=====================
+
+.. image:: img/nav_navmesh_links.png
+
+NavigationLinks are used to connect navmesh polygons from :ref:`NavigationRegion2D<class_NavigationRegion2D>` 
+and :ref:`NavigationRegion3D<class_NavigationRegion3D>` over arbitrary distances for pathfinding.
+
+NavigationLinks are also used to consider movement shortcuts in pathfinding available through 
+interacting with gameplay objects e.g. ladders, jump pads or teleports.
+
+2D and 3D versions of NavigationJumplinks nodes are available as 
+:ref:`NavigationLink2D<class_NavigationLink2D>` and 
+:ref:`NavigationLink3D<class_NavigationLink3D>` respectively.
+
+Different NavigationRegions can connect their navmeshes without the need for a NavigationLink 
+as long as they are within navigation map ``edge_connection_margin`` and have compatible ``navigation_layers``.
+As soon as the distance becomes too large, building valid connections becomes a problem - a problem that NavigationLinks can solve.
+
+See :ref:`doc_navigation_using_navigationregions` to learn more about the use of navigation regions.
+See :ref:`doc_navigation_connecting_navmesh` to learn more about how to connect navigation meshes.
+
+.. image:: img/nav_link_properties.png
+
+NavigationLinks share many properties with NavigationRegions like ``navigation_layers``.
+NavigationLinks add a single edge connection between two positions over an arbitrary distance 
```

Comment:
```suggestion
NavigationLinks add a single connection between two positions over an arbitrary distance 
```

Consider dropping the edge here, since it's a pretty minor implementation detail that new comers might not understand.